### PR TITLE
47 Populate Mood Tabs

### LINKED ISSUE #47: Populate Mood Tabs
**__Task Description__**
- Map all moods into tabs which will locate under Trending Movies section
   - Mood object can be invoked using `movieMoods` from `./data`
   - Fetch movies by mood with URL `/api/fetch-movie-mood?moodId=${mood}&pageId=${pageNumber}`
- There are 11 moods to choose from. Feel free to add or delete!

### LINKED ISSUE #57: Add pagination under `MovieRow.js` section
**__Task Description__**
Describe the task to be completed.

**__Technical Considerations__**
Include any technical considerations including architecture (e.g. API), required libraries, etc.

**__Additional Considerations__**
Any supplemental information including unresolved questions, links to external resources, screenshots, etc.

### PR DESCRIPTION
**Pull request by kacydodev**
https://github.com/chingu-voyages/v41-geckos-team-18/issues/47

- Mapped out mood tabs and pagination pages
- When switch to new mood tab, always show first page
- NOT mobile friendly atm

## ⚠️ Current Issues 
- `<Tab>` doesn't work on mobile view - partially hidden on small screen
- Some movie posters are missing - Needs poster placeholder
- When switch tab or page, in-between loading, tab panel height increased and temporary hides pagination element

### CODE REVIEW

File: movie-app/src/components/MoodGallery/MoodContent.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  const handleOpen = (movieData) => {
+    setOpen(true);
+    setSelectedMovie(movieData);
+  };
+  const handleClose = () => setOpen(false);
+
+  return (
+    <>
+      {isLoading ? (
+        <p>Loading...</p>
```
Comment (sukyungdev) [inline movie-app/src/components/MoodGallery/MoodContent.js:29]: I think this is the reason for the increase in the height of the tab panel.
How about using Position Absolute?

**Review by sukyungdev — APPROVED** [review]
Great! I don't think there are any errors other than three.

**Comment by sukyungdev** [comment]
How about arranging different way on mobile?
Is there a way you would like to do it?

File: movie-app/src/components/MoodGallery/MoodContent.js (hunk 1 above)
Comment (kacydodev) [reply movie-app/src/components/MoodGallery/MoodContent.js:29]: Ohhh I think you are right!! Thank you thank you thank you 🙏

**Comment by kacydodev** [comment]
> 

I'll merge up and figure it out on the next iteration 😅

**Pull request merged**